starwars/src/App.js

```diff
@@ -2,43 +2,43 @@ import './App.css';
 import axios from 'axios';
 import React, { useEffect, useState } from "react";
 
-
 function App() {
   const [planets, setPlanets] = useState({})
   let [id, setId] = useState(1)
 
-  const BASE_URL = 'https://swapi.dev/api'
+  const BASE_URL = 'https://swapi.dev/planets'
   // then and catch version
   // change the url to see what else you can look at, like /planets or /starships
   const requestPlanetsData = () => {
     axios
       .get(`${BASE_URL}/people/${id}`)
       .then(res => {
-        // try console logging res.data
-        console.log(res)
+        // try console logging res.data (Did this)
+        console.log(res.data)
         return setPlanets(res.data)
       })
       .catch(err => {
         console.error(err)
       })
   }
-  // async version, uncomment to try out
-  // const requestCharacterData = async () => {
-  //   try {
-  //     let res = await axios.get(`${BASE_URL}/people/${id}`);
-  //     let { data } = res;
-  //     return setCharacter(data)
-  //   } catch(err) {
-  //     console.error(err)
-  //   }
-  // };
+
```

Comment:
Hi zach! Great work!

Please comment the .then and .catch version and use only the async await. Where is the setPlanet coming from? I only see the setPlanets hook here.